src/Illuminate/Contracts/Cookie/Factory.php

```diff
@@ -10,8 +10,8 @@
      * @param  string  $name
      * @param  string  $value
      * @param  int     $minutes
-     * @param  string  $path
-     * @param  string  $domain
+     * @param  string|null  $path
```

Comment:
Shouldn't these be aligned? :)

Reply:
Others parameters in this and others methods are not aligned so I won't align those if not really necessary.